generation/new_client/README.md

```diff
@@ -5,17 +5,40 @@ google-cloud-java monorepo.
 
 **This tool is for repository maintainers only. Not for library users.**
 
-## Prerequisites
-
-This section is only needed for the first run of this script. If it's already
-done, go to "Run client generation script" section.
-
-
-### Environment
-
-Use Linux environment.
-
-Install Docker.
+## Run via github action
+
+You can save the time to setup the environment by calling the
```

Comment:
> You can save the time to setup the environment by calling the

This reads as if this is a good _option_ for the reader. This is not the option. Would you rewrite this is the way for the reader to follow?